erizo/src/erizo/rtp/SenderBandwidthEstimantionHandler.cpp

```diff
@@ -128,6 +144,34 @@ void SenderBandwidthEstimationHandler::read(Context *ctx, std::shared_ptr<DataPa
   ctx->fireRead(std::move(packet));
 }
 
+void SenderBandwidthEstimationHandler::updateReceiverBlockFromList() {
+  if (rr_delay_data_.size() < max_rr_delay_data_size_) {
+    return;
+  }
+  // TODO(pedro) Implement alternative when there are no REMBs
+  if (received_remb_) {
+    uint32_t total_packets_lost = 0;
+    uint64_t avg_delay = 0;
+    uint32_t rr_delay_data_size = rr_delay_data_.size();
+    int64_t now_ms = ClockUtils::timePointToMs(clock_->now());
+    std::for_each(rr_delay_data_.begin(), rr_delay_data_.end(),
+        [&avg_delay, &total_packets_lost, rr_delay_data_size] (const std::shared_ptr<RrDelayData> &rr_info) {
+          total_packets_lost += rr_info->packets_lost;
+          avg_delay += rr_info->delay / rr_delay_data_size;
+    });
+    if (period_packets_sent_ > 0) {
+      uint32_t fraction_lost = total_packets_lost * 255 / period_packets_sent_;
```

Comment:
I think you can't calculate the fraction lost in a period from the packets lost you get in a Receiver Report. You would have to keep track of the differences in packet loss between two consecutive Receiver Reports since it indicates the cumulative lost packets.

Reply:
good catch!